index.html

```diff
@@ -21,7 +21,7 @@
     <div class="gradient">
        <div class="welcome">
          <img class="welcomeImage" src="images\isTarget.jpg" />
-      </div>
+    </div>
```

Comment:
What is the reason of this change?
Apart from that I have a feeling that you have one div not closed here
Why did not you reformat html files earlier?